Refit/RequestBuilderImplementation.cs

```diff
@@ -38,7 +38,12 @@ public RequestBuilderImplementation(Type targetInterface, RefitSettings refitSet
                     var hasHttpMethod = attrs.OfType<HttpMethodAttribute>().Any();
                     if (!hasHttpMethod) return Enumerable.Empty<RestMethodInfo>();
 
-                    return EnumerableEx.Return(new RestMethodInfo(targetInterface, x, settings));
+                    var unsuccessfulFilters = attrs.OfType<UnsuccessfulResponseFilterAttribute>();
+
+                    return EnumerableEx.Return(new RestMethodInfo(targetInterface, x, settings)
+                    {
+                        UnsuccessfulResponseFilters = unsuccessfulFilters.Any() ? unsuccessfulFilters.OrderBy(f => f.Order).ToArray() : null
```

Comment:
Multiple enumeration of `IEnumerable`

Reply:
Here, `.Any()` checks only for the first item, and is better than allocating memory with `ToList` or `ToArray`. Considering the source is already an array, `.Any()` is amost free, and it only iterates the enumerable if it really needs to.